polaris/utils/errors.py

```diff
@@ -60,7 +57,6 @@ def __init__(self, response: Response | None = None):
             "Note: If you can confirm that you are authorized to perform this action, "
             "please call 'polaris login --overwrite' and try again. If the issue persists, please reach out to the Polaris team for support."
         )
-        message = self.format(message, [self.BOLD, self.YELLOW])
```

Comment:
Have we lost the formatting of our error messages now? If so, I would've personally rathered the dubious import of the `PolarisChecksumError` in the `ChecksumMixin` class to preserve their styling while we create a separate ticket to more properly address it. Not a blocker though!

Reply:
We indeed lost the text formatting for the errors. I personally think that's okay. We're not using the formatting consistently for all errors and I feel like the whole "error" system (and logging, and module structure) is something we may want to address in the future.

Reply:
Sounds good. I think we should address it when we get some time. There should be a ticket regarding it floating around somewhere!